public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,192 @@
+# Deploy on Heroku
+
+### Pre-requirements
+
+- You must have an [account]((https://signup.heroku.com/login)) on heroku
+- You should have [git](https://git-scm.com/) installed
+- Your application should compile without error using dmd (`dub build --build=release`). 
+
+
+### 1 : Setup the app
+
+The first thing we need to do before deploying our app to the cloud is binding our app's port to heroku's one. 
```

Comment:
As @wilzbach pointed out better not use we here. Just write "The first thing to do is.."